pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -75,17 +75,15 @@ type ReconcileRpaasInstance struct {
 func (r *ReconcileRpaasInstance) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling RpaasInstance")
-
-	instance := &extensionsv1alpha1.RpaasInstance{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	manager := rpaas.NewK8S(rpaas.K8SOptions{
+		Cli: r.client,
+		Ctx: context.Background(),
+	})
+	instance, err := manager.GetInstance(request.NamespacedName.Name)
 	if err != nil {
-		if errors.IsNotFound(err) {
```

Comment:
Is it possible that removing this condition will cause Reconcile to be called continuously for removed RpaasInstances? Also I'm not entirely sure we want to use the manager here.

Reply:
Hummm.... I'll test this scenario and tell you what happens when we remove a RpaasInstance CR.

About using the `RpaasManager` here, I just want to avoid code duplication for getting the RpaasInstances. I really think isn't a big problem.

Reply:
Yeah, you're right. The Reconcile is being continuously called after a remove of a RpaasInstance. I'll change that too. Thanks!